apps/ejabberd/src/mongoose_commands.erl

```diff
@@ -117,6 +118,7 @@
           args = [] :: [argspec()],                   %% this is both for introspection and type check on call
           caller_pos :: integer(),                    %% internal use
           identifiers = [] :: [atom()],               %% resource identifiers, a subset of args
+          queryparams = [] :: [atom()],               %% query parameters, a subset of args
```

Comment:
About your documentation request - those modules are enabled by default, do we need to put it anywhere else?